lasagne/layers/cuda_convnet.py

```diff
@@ -13,7 +13,16 @@
 from .pool import pool_output_length
 from ..utils import as_tuple
 
-from theano.sandbox.cuda.basic_ops import gpu_contiguous
+# check if Theano's old GPU backend is available and in use
+try:
+    from theano.sandbox.cuda.basic_ops import gpu_contiguous
+except:
```

Comment:
Should be `except ImportError`. A naked `except:` will even catch a CTRL+C by the user.